lib/routers/profile/pil.js

```diff
@@ -64,6 +65,14 @@ const checkEstablishment = (req, res, next) => {
   if (req.pil.establishmentId !== req.establishment.id) {
     return next(new BadRequestError());
   }
+
+  if (req.params.action === 'transfer') {
+    const toEstablishmentId = get(req.body.data, 'establishment.to.id');
+    if (!req.user.profile.establishments.find(e => e.id === toEstablishmentId)) {
+      next(new BadRequestError('Can only transfer a PIL to establishments the user is associated with'));
```

Comment:
```suggestion
      return next(new BadRequestError('Can only transfer a PIL to establishments the user is associated with'));
```

Otherwise you'll get a `Cannot set headers after they are sent` error